library/std/src/sys/windows/c/windows_sys.lst

```diff
@@ -0,0 +1,2587 @@
+Windows.Wdk.Storage.FileSystem.NTCREATEFILE_CREATE_DISPOSITION
```

Comment:
This file seems like a bit of a pain to maintain, but so it goes. At the very least, we should probably ensure this file is sorted (and ideally check it in tidy).

Reply:
Sorting sounds good.

My feeling is this list should probably encompass a fair amount of "unused but might be used" things. So hopefully it won't need to be touched too often. Admittedly there is a balance between size and convenience but I think we don't need to be too precise about it..

Reply:
Is there any description how this list created?

Reply:
Originally? Me hacking on windows-rs, some ripgrep of `c.rs` and some manual fixes, unfortunately. Though I hope adding things to it is simpler now. Just add the things you use (using [docs.rs/windows-sys](https://docs.rs/windows-sys/latest/windows_sys/) as a reference)

Reply:
For example, from ~200 DNS_ERROR_* consts there currently used only `DNS_ERROR_RECORD_TIMED_OUT`; from ~400 `ERROR_DS_*` used only `ERROR_DS_TIMELIMIT_EXCEEDED` (but a bunch of them was defined in old errors.rs, yes), so i'm trying to understand if that big list really necessary.

Reply:
Necessary? No, probably not. But, as you say, the old errors.rs included a big list and also I wanted to avoid people having to add or remove error constants as much as possible. I could reduce it but then I'd need to make more of a judgement on what to include or not. And I didn't think it was worth it because I suspect the extra consts won't make much difference to compile times.